tc_aws/aws/storage.py

```diff
@@ -38,6 +38,10 @@ def set(self, bytes, abspath):
         file_key=Key(self.storage)
         file_key.key=abspath
 
+        if self.context.config.S3_STORE_METADATA:
```

Comment:
Could you rather use the ``self.context.config.get(...)`` syntax? In order to harmonize with the rest of the code :)